lib/marqeta/api_caller.rb

```diff
@@ -0,0 +1,43 @@
+require 'json'
+require 'rest-client'
+
+module Marqeta
+  class ApiCaller
+    def initialize(endpoint)
+      @endpoint = endpoint
+    end
+
+    def get
+      logger.info("GET: #{endpoint}")
+      response_hash = JSON.parse(resource.get)
+      logger.info("Response: #{response_hash.to_json}")
```

Comment:
I see a little problem here: If the resource can't be parsed as valid JSON, it'll fail before it's able to log this.

Also, by parsing it then turning it back into a string, there's potential for the logs to be misleading, since encoding/decoding isn't always 'symmetrical'.

I wonder if it should log the raw response before attempting to parse it?

Reply:
ok yeah good idea. I'll change that.